database/user.js

```diff
@@ -0,0 +1,36 @@
+const pgp = require( 'pg-promise' )()
+const db = pgp( process.env.DATABASE_URL )
+const bcrypt = require( 'bcrypt-nodejs' )
+
+const FIND_BY_EMAIL = `SELECT * FROM users WHERE email=$1`
```

Comment:
Knex's functions sometimes arent as reliable as just using raw SQL. I've run into a couple of really wired bugs when using knex's query builder functions.